### PR TITLE
Loading screen improvements

### PR DESCRIPTION
**Pull request by jonbell-lot23**
**The problem we're trying to solve**

For NUX v2, we'd like to educate people about app features. One place to educate is our loading screen, so we've added some helpful tips to our standard loading messages.

Done:

- [x] Removed the bubbles in the background
- [x] Added three tips for focus mode, command palette, and print statements
- [x] Polished the visuals a bit
- [x] Added randomised messages (previously we weren't randomising at all!)

Up next:

- [x] Make sure my code is good (especially the way I'm randomising)
- [x] Socialise the work

Screenshot:

image

### CODE REVIEW

File: src/ui/components/shared/LoadingTips.tsx [hunk 1]
```diff
@@ -18,21 +19,35 @@ const TIPS = [
...
+    description:
+      "Print statements might be Replay's best feature! Try clicking on a line of code in the editor.",
+    icon: LightbulbIcon,
+  },
 ] as const;
 
 export const LoadingTips: FC = () => {
-  const [currentTipIdx, setCurrentTipIdx] = useState(0);
+  const rand = Math.floor(Math.random() * (TIPS.length));
+  console.log(rand);
```
Comment (jaril) [inline src/ui/components/shared/LoadingTips.tsx:49]: Remove print statement

**Review by jaril — APPROVED** [review]

File: src/ui/components/shared/LoadingTips.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/shared/LoadingTips.tsx:49]: D'oh/duh

**Comment by jonbell-lot23** [comment]
Ready for review

**Comment by ryanjduffy** [comment]
@jonbell-lot23 - Do you mean for the login page to look this way with the change? It also uses the `BubbleBackground` with the dark theme and looks a bit off to me.

image

**Comment by jonbell-lot23** [comment]
@ryanjduffy Darn, didn't mean for that to do that. I think it's an issue with some changes we made with dark/light theme and the way we do onboarding.

**Comment by jonbell-lot23** [comment]
Going to put this on hold for now. Will be chatting with Ryan next week.

**Comment by jonbell-lot23** [comment]
Ok @ryanjduffy I think I/we fixed:

1. The flickering message thing (thanks for your help!)
2. Dark/light theme logo for login screen

Where'd we end up? Is this ready to go, or were there other things we were supposed to deal with?

**Review by ryanjduffy — APPROVED** [review]
Cleaned up a few things but LGTM otherwise!

**Pull request merged**